extensions/HuggingFace/python/src/aiconfig_extension_hugging_face/local_inference/util.py

```diff
@@ -0,0 +1,28 @@
+from typing import TYPE_CHECKING
+from aiconfig import ParameterizedModelParser
+from aiconfig.schema import Prompt
+
+
+# Circular Dependency Type Hints
+if TYPE_CHECKING:
+    from aiconfig import AIConfigRuntime
+
+
+def get_hf_model(aiconfig: "AIConfigRuntime", prompt: Prompt, model_parser: ParameterizedModelParser) -> str | None:
```

Comment:
If possible it's much better to untangle the circle. Do we even have a circle in this case?